dev/VSIX/Shared/WizardImplementation.cs

```diff
@@ -90,7 +103,7 @@ await ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 // Start the threaded wait dialog
                 if (_waitDialog != null)
                 {
-                    _waitDialog.StartWaitDialog(null, "Installing NuGet packages into project...", null, null, "Operation in progress...", 0, false, true);
+                    _waitDialog.StartWaitDialog(null, Resources._1044, null, null, Resources._1045, 0, false, true);
```

Comment:
I understand the numbers are how they are defined in the resource files. But why do we choose to keep using number here instead of meaningful variable names for the `Resources` object?